exercises/practice/simple-linked-list/.docs/instructions.append.md

```diff
@@ -1,16 +1,87 @@
 # Instructions append
 
+## How this Exercise is Structured in Python
+
+While `stacks` and `queues` can be implemented using `lists`, `collections.deque`, `queue.LifoQueue`, and `multiprocessing.Queue`, this exercise expects a ["Last in, First Out" (`LIFO`) stack][Baeldung: The Stack Data Structure] (_interactive example [here][LIFO Stack]_) using a _custom-made_ [singly linked list][singly linked list].
+
+<br>
+
+_push and pop happen here_
```

Comment:
This line indicates that both happen here, but it only addresses one…

Reply:
It addresses neither.  The representation is meant to show the `next` pointer between nodes - not the push nor pop actions. 

Unfortunately, I am limited by the ASCII art here.  But if you can find a way to cleanly show push and pop actions here within how it is written, I am open to updating it.  Otherwise, we need to either embed a full diagram (_which then becomes difficult to read when the exercise is downloaded via CLI_), or remove this ascii art as an example.

The confusing point here for many is that a `stack` underpinned by a Linked List has the `head` on the "left" -- not that push and pop happen _at_ the head.  See discussions under #3009 for why this was included in the first place. 

For a diagram, we could custom-make a variant on the below, but I'd rather not.

![2022-05-07_11-06-51](https://user-images.githubusercontent.com/5923094/167266534-89751b12-2cbe-4bca-b4b0-8de39cd4143a.png)

Reply:
I saw the pop and shift happens here, and the vertical line which I thought was indicating the exact position.  So that was the confusing part, and then only the one vertical line, which I thought the "and" part was missing from either, or the "and" was simply extraneous.

That that I know it shows neither... a bit more confusing to read, I think.

I do agree with the hard to read in the terminal if we have a graphic representation.  I do see it as a mermaid friendly problem, but how many people can read mermaid charts in the terminal as plain text and see what it is meant to depict?  Not sure.

Thinking about what would look good in markdown as rendered and as plain text…

I will discard my review until I can think of a solution for that.